docs/content/docs.md

```diff
@@ -433,6 +433,14 @@ Normally rclone would skip any files that have the same
 modification time and are the same size (or have the same checksum if
 using `--checksum`).
 
+### --immutable ###
+
+Treat source and destination files as immutable and disallow
+modification.  Fail if existing files have been modified.
+
+This can be useful as an additional layer of protection for
+immutable or append-only data sets.
+
```

Comment:
Could you put a bit more in here please.

Note that the new files are transferred from source to destination but existing files on the destination are never changed.

You should note that it only works for sync/copy/move too.

Also a bit more on use cases would be great!